ddsrouter_test/compose/test_cases/stress/single_router/compose.yml

```diff
@@ -0,0 +1,197 @@
+#  │ │ publisher_5      │ ││           │┼─► (local)           │ │
+#  │ │ ...              │ ││           ││ └───────────────────┘ │
+#  │ │ publisher_8      ├─►│ ddsrouter ││                       │
+#  │ │ (local)          │ ││           ││ ┌───────────────────┐ │
+#  │ └──────────────────┘ ││  (local)  ││ │ subscriber_3      │ │
+#  │                      ││           │┼─► (local)           │ │
+#  │ ┌──────────────────┐ ││           ││ └───────────────────┘ │
+#  │ │ publisher_9      │ ││           ││                       │
+#  │ │ ...              │ ││           ││ ┌───────────────────┐ │
+#  │ │ publisher_12     ├─►│           ││ │subscriber_reliable│ │
+#  │ │ (local)          │ ││           │┼─► (local)           │ │
+#  │ └──────────────────┘ ││           ││ └───────────────────┘ │
+#  │                      ││           ││                       │
+#  │ ┌──────────────────┐ ││           ││                       │
+#  │ │publisher_reliable├─►│           ││                       │
+#  │ │ (local)          │ └┼───────────┼┘                       │
+#  │ └──────────────────┘  │           │                        │
+#  │                       │           └────────────────────────┘
+#  └───────────────────────┘
+
+
+
+
+
+
+
+services:
+
+  # ROUTER
+  ddsrouter:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: ddsrouter
+    networks:
+      - std_net
+    volumes:
+      - ./ddsrouter.yaml:/config.yaml
+    command: ddsrouter -c /config.yaml --timeout 15
+
+
+ # SUBSCRIBERS
+  subscriber_1:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: subscriber_1
+    networks:
+      - std_net
+    volumes:
+      - ../../../scripts:/scripts
+    command: python3 /scripts/execute_and_validate_subscriber.py --exe install/AdvancedConfigurationExample/examples/cpp/dds/AdvancedConfigurationExample/AdvancedConfigurationExample --samples 10 --timeout 10 --allow-duplicates -1 --args "--samples 10 --domain 105 --topic LargeDataTopic_1"
+
+  subscriber_2:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: subscriber_2
+    networks:
+      - std_net
+    volumes:
+      - ../../../scripts:/scripts
+    command: python3 /scripts/execute_and_validate_subscriber.py --exe install/AdvancedConfigurationExample/examples/cpp/dds/AdvancedConfigurationExample/AdvancedConfigurationExample --samples 10 --timeout 10 --allow-duplicates -1 --args "--samples 10 --domain 105 --topic LargeDataTopic_2"
+
+  subscriber_3:
+    image: ${DDSROUTER_COMPOSE_TEST_DOCKER_IMAGE}
+    container_name: subscriber_3
+    networks:
+      - std_net
+    volumes:
+      - ../../../scripts:/scripts
+    command: python3 /scripts/execute_and_validate_subscriber.py --exe install/AdvancedConfigurationExample/examples/cpp/dds/AdvancedConfigurationExample/AdvancedConfigurationExample --samples 10 --timeout 10 --allow-duplicates -1 --args "--samples 10 --domain 105 --topic LargeDataTopic_3"
+
+  subscriber_reliable:
```

Comment:
I would have the reliable subscriber and the reliable publisher together, right below the router. They are the most important part of the test and they should be easily spotted.

Reply:
I think that to be consistent it's better if we run before the subscribers and then the publishers, also because most of the subscribers are best effort so if we run them after the publisher there is a risk that they will not receive all the samples. Anyway both of them are the first ones of the each list.